parachain/Cargo.toml

```diff
@@ -95,12 +95,14 @@ substrate-build-script-utils = { git = "https://github.com/paritytech/substrate.
 
 [features]
 default = [ "snowblink-native", "snowbase-native" ]
-runtime-benchmarks = [
-    "snowbridge-runtime/runtime-benchmarks",
-    "snowblink-runtime/runtime-benchmarks",
-    "snowbase-runtime/runtime-benchmarks",
-    "polkadot-service/runtime-benchmarks"
-]
+
+snowbridge-runtime-benchmarks = [ "snowbridge-runtime/runtime-benchmarks" ]
+snowblink-runtime-benchmarks = [ "snowblink-runtime/runtime-benchmarks" ]
+snowbase-runtime-benchmarks = [ "snowbase-runtime/runtime-benchmarks" ]
+polkadot-runtime-benchmarks = [ "polkadot-service/runtime-benchmarks" ]
+
+runtime-benchmarks = []
```

Comment:
This change is to prevent all the runtimes from being included at once, and doing so enabling the `minimal` feature for all runtimes, which then breaks the `snowblink` and `snowbridge` benchmarks for the beacon pallet.

The downside is that if we want to run benchmarks for a runtime, we need to include 2 features:

`features = "snowblink-runtime-benchmarks,runtime-benchmarks"`

We cannot just have the runtime benchmark feature (i.e. `snowblink-runtime-benchmarks`) because Substrate checks if the pallet was compiled with feature `runtime-benchmarks`, so we need both.

Reply:
Ok, yeah I see what you mean. Not ideal but I don't see another way around it.